performance_manager/lib/l1_full_trip_events.py

```diff
@@ -433,7 +447,13 @@ def insert_new_events(db_manager: DatabaseManager) -> None:
                 FullTripEvents.hash == TempFullTripEvents.hash,
                 isouter=True,
             )
-            .where((FullTripEvents.hash.is_(None))),
+            .where(
+                (FullTripEvents.hash.is_(None))
+                & sa.or_(
+                    TempFullTripEvents.fk_vp_moving_event.isnot(None),
+                    TempFullTripEvents.fk_vp_stopped_event.isnot(None),
+                )
+            ),
```

Comment:
This makes sure that there are no `FullTripEvent` records with just a `TripUpdate` key.